src/components/ArticleBlock.jsx

```diff
@@ -0,0 +1,21 @@
+function ArticleBlock({ article }) {
+  const date = new Date(article.created_at);
+  const format = { year: "numeric", month: "long", day: "numeric" };
+  const formattedDate = date.toLocaleDateString("en-US", format);
+
+  return (
+    <>
+      <li className="article-block">
+        <img src={article.article_img_url} alt={article.title} />
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api, as well as the votes and comment counts for each article